contrib/face_recognizer.go

```diff
@@ -24,6 +24,26 @@ type FaceRecognizer interface {
 	Close() error
 }
 
+type IBasicFaceRecognizer interface {
```

Comment:
I think a better name for the interface would be just `BasicFaceRecognizer`

Reply:
See above additional comment.

Reply:
Yes! I forgot to change them back. Was trying to implement the interfaces more go-stylish.